libraries/ms-drivers/inc/mcp23008_gpio_expander.h

```diff
@@ -0,0 +1,56 @@
+#pragma once
+// GPIO HAL interface for MCP23008 GPIO expander
```

Comment:
Apply all my reviews for the `gpio_expander` here, then we'll close that pull request and continue on this :)